src/main/java/org/cascadebot/cascadebot/events/CommandListener.java

```diff
@@ -185,11 +185,13 @@ private void processCommands(GuildMessageReceivedEvent event, GuildData guildDat
             dispatchCommand(cmd, context);
         }
 
-        //TODO maybe add setting for this?
-        if (guildData.getTagInfo().containsKey(trigger)) {
-            Tag tag = guildData.getTag(trigger);
+        if(guildData.getSettings().willAllowTagCommands()) {
+            if (guildData.getTagInfo().containsKey(trigger)) {
+                Tag tag = guildData.getTag(trigger);
 
-            context.reply(tag.formatTag(context)); //Are we adding perms to tags? If so i need to implement that. - Ryan
+                context.reply(tag.formatTag(context)); //Are we adding perms to tags? If so i need to implement that. - Ryan
+                CascadeBot.LOGGER.info("Tag {} executed by {} with args {}", trigger, context.getUser().getAsTag(), Arrays.toString(context.getArgs()));
```

Comment:
I'm honestly tempted to say that we should leave this at debug level

Reply:
Bump

Reply:
I doubt tags will be used more then commands and we're already logging all commands used.